apps/studio-client/src/main/joo/com/coremedia/labs/translation/gcc/studio/GccStudioPluginBase.as

```diff
@@ -109,33 +107,38 @@ public class GccStudioPluginBase extends StudioPlugin {
     };
   }
 
-  internal static function getTranslationCancelToolBarButtonGCC(workflowObjectVE:ValueExpression, panel:Panel):IconButton {
-
-    var process:Process = getProcess(workflowObjectVE.getValue());
-
+  internal static function getTranslationCancelToolBarButtonGCC(workflowObjectVE:ValueExpression, panel:GridPanel):IconButton {
     var buttonCfg:IconButton = IconButton({});
     buttonCfg.itemId = "cancelButtonID";
+    buttonCfg.disabled = !isButtonEnabled(workflowObjectVE);
     buttonCfg.tooltip = ResourceManager.getInstance().getString('com.coremedia.labs.translation.gcc.studio.GccProcessDefinitions', 'Action_Tooltip_Cancel_Process');
     buttonCfg.iconCls = ResourceManager.getInstance().getString('com.coremedia.icons.CoreIcons', 'remove');
-    buttonCfg.handler = function ():void {
-      var processName:String = process.getProperties().get(SUBJECT_VARIABLE_NAME) as String;
-      var messageText:String = ResourceManager.getInstance().getString('com.coremedia.labs.translation.gcc.studio.GccProcessDefinitions', 'confirm_cancellation', [processName]);
 
+    var cancelButton:IconButton = new IconButton(buttonCfg);
+
+    workflowObjectVE.addChangeListener(function(woVE:ValueExpression):void {
+      cancelButton.setDisabled(!isButtonEnabled(woVE));
+    });
+
+    cancelButton.setHandler(function ():void {
+      var messageText:String = ResourceManager.getInstance().getString('com.coremedia.labs.translation.gcc.studio.GccProcessDefinitions', 'confirm_cancellation');
       MessageBoxUtil.showPrompt("Cancel Workflow", messageText, function (result:String):void {
```

Comment:
Not a regression - but we may want to add a localized title to the message box here, too.